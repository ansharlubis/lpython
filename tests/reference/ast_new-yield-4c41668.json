{
    "basename": "ast_new-yield-4c41668",
    "cmd": "lpython --show-ast --new-parser --no-color {infile} -o {outfile}",
    "infile": "tests/parser/yield.py",
    "infile_hash": "959f36be85d0fd6430404508594282308b6ece2686a8e385f8cf9d48",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "ast_new-yield-4c41668.stdout",
    "stdout_hash": "be852fd40c75ee6d3f3fcefafb2ee98efca767e8ca54bf0b5056b617",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}