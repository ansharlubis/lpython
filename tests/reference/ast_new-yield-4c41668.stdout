(Module [(FunctionDef func ([] [] [] [] [] [] []) [(AnnAssign (Name x Store) (Subscript (Name list Load) (Name i32 Load) Load) () 1) (Assign [(Name x Store)] (List [(ConstantInt 1 ()) (ConstantInt 2 ()) (ConstantInt 3 ())] Load) ()) (AnnAssign (Name i Store) (Name i32 Load) () 1) (For (Name i Store) (Name x Load) [(Expr (Yield (Name i Load)))] [] ()) (For (Name i Store) (Name x Load) [(Expr (Yield (Name i Load)))] [] ()) (For (Name i Store) (Name x Load) [(Expr (Yield (Tuple [] Load)))] [] ()) (For (Name _ Store) (Name x Load) [(Expr (Yield ()))] [] ())] [] () ())] [])
